docs/src/grigoriev/grigoriev.md

```diff
@@ -63,47 +69,46 @@ Let's generate a triangulated grid located at the appropriate coordinates:
 # Specify a resolution:
 RES_Y = (MAX_Y-MIN_Y)/30.0
 RES_X = RES_Y
+
 # Generate the mesh with appropriate dimensions and resolution:
-s′ = triangulated_grid(
-                       MAX_X-MIN_X, MAX_Y-MIN_Y,
-                       RES_X, RES_Y, Point3D)
+s = triangulated_grid(MAX_X-MIN_X, MAX_Y-MIN_Y, RES_X, RES_Y, Point3D)
+
 # Shift it into place:
-s′[:point] = map(x -> x + Point3D(MIN_X, MIN_Y, 0), s′[:point])
-s = EmbeddedDeltaDualComplex2D{Bool, Float64, Point3D}(s′)
-subdivide_duals!(s, Barycenter())
+s[:point] = map(x -> x + Point3D(MIN_X, MIN_Y, 0), s[:point])
+sd = EmbeddedDeltaDualComplex2D{Bool, Float64, Point3D}(s)
+subdivide_duals!(sd, Barycenter())
 
 fig = Figure()
 ax = CairoMakie.Axis(fig[1,1])
-wf = wireframe!(ax, s)
+wf = wireframe!(ax, sd)
 display(fig)
 ```
 
-The coordinates of a vertex are stored in `s[:point]`. Let's use our interpolator to assign ice thickness values to each vertex in the mesh:
+The coordinates of a vertex are stored in `sd[:point]`. Let's use our interpolator to assign ice thickness values to each vertex in the mesh:
 
 ``` @example DEC
 # These are the values used by the CISM benchmark:
 n = 3
 ρ = 910
 g = 9.8101
-A = fill(1e-16, ne(s))
+A = fill(1e-16, ne(sd))
 
-h₀ = map(s[:point]) do (x,y,_)
+h₀ = map(sd[:point]) do (x,y,_)
   tif_val = ice_interp(x,y)
   # Accommodate for the -∞'s that encode "no ice".
   tif_val < 0.0 ? 0.0 : tif_val
 end
 
 # Store these values to be passed to the solver.
 u₀ = ComponentArray(h=h₀, stress_A=A)
-constants_and_parameters = (
-  n = n,
-  stress_ρ = ρ,
-  stress_g = g,
-  stress_A = A)
+constants_and_parameters = (n = n, stress_ρ = ρ,
+                            stress_g = g, stress_A = A)
+nothing # hide
 ```
 
-# Defining and Composing Models
-For exposition on this Halfar Decapode, see our [Glacial Flow](https://algebraicjulia.github.io/Decapodes.jl/dev/ice_dynamics) docs page. You can skip ahead to the next section.
+## Defining and Composing Models
+
+For exposition on this Halfar Decapode, see our [Glacial Flow](../ice_dynamics.md) docs page. Otherwise, you may skip ahead to the next section.
```

Comment:
Does Documenter replace this with a link to the render docs page? If not put the url back

Reply:
Yes it does, and we should definitely be using relative links.